src/components/Hamburger/Hamburger.tsx

```diff
@@ -36,6 +37,14 @@ export interface HamburgerProps
    */
   labelInActive: string;
   'data-testid'?: string;
+  /**
+   * Additional data that is dispatched when opening hamburger.
+   */
+  trackingOpen?: TrackingProps;
+  /**
+   * Additional data that is dispatched with closing hamburger.
+   */
+  trackingClose?: TrackingProps;
```

Comment:
I'd prefer a single `tracking` prop here to keep it consistent with other components. If a user wants to send different event data when the Hamburger is open/closed, they can do it like so:

```ts
const [isActive, setActive] = useState(false);

return (
  <Hamburger 
    isActive={isActive}
    tracking={isActive ? { label: 'close-hamburger' } : { label: 'open-hamburger' }}
  />
);